UI/Selenium/Helper/Helper.cs

```diff
@@ -24,13 +25,13 @@ public static IWebElement Find(this IWebDriver driver, By by,int timeoutSeconds=
 
                     if (element.Displayed && element.Enabled)
                     {
-                        Logger.Info(" The following element has been found " + element);
+                        Logger.InfoWithDate("The following element has been found " + element);
                         return element;
                     }
                 }
                 catch (NoSuchElementException e)
                 {
-                    Logger.Error(" The following error has occourred" + e);
+                    Logger.Error("The following error has occourred" + e);
```

Comment:
Can you include the stacktrace in the logger.  And, I need to see more detail, what element does not exist? (Element name and page name.)

Reply:
Stacktrace has been included along with the Exception